### PR TITLE
Refactor cobra code

### PR DESCRIPTION
**Pull request by Marton6**
This PR splits up `env.go` into `app.go`, `argument.go` and `flag.go` so the code can be managed more easily.

A couple typos from comments have also been fixed.

### CODE REVIEW

File: internals/cli/argument.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+func shortDur(d *time.Duration) string {
+	s := d.String()
+	if strings.HasSuffix(s, "m0s") {
+		s = s[:len(s)-2]
+	}
+	if strings.HasSuffix(s, "h0m") {
+		s = s[:len(s)-2]
+	}
+	return s
+}
```
Comment (edif2008) [inline internals/cli/argument.go:103]: This is used by `DurationVarP` and `DurationVar` from `flag.go`. Therefore, it should be moved there.

**Review by edif2008 — COMMENTED** [review]
Nice division of the 4 custom objects that we made.
One small adjustment, and this can be merged.

File: internals/cli/argument.go (hunk 1 above)
Comment (Marton6) [reply internals/cli/argument.go:103]: Oh, that's right

Comment (Marton6) [reply internals/cli/argument.go]: Fixed

**Review by edif2008 — APPROVED** [review]

**Pull request merged**